raiden/tests/scenarios/pfs5_too_low_capacity.yaml

```diff
@@ -77,8 +78,9 @@ scenario:
 
             # There is not enough capacity through node4, so path [0, 1, 2, 3] 
             # should be used instead.
+            - wait_blocks: 2
             - transfer: {from: 0, to: 3, amount: 1_000_000_000_000_000, expected_http_status: 200}
-
+            - wait_blocks: 2
```

Comment:
This is where it makes sense to wait.

Reply:
Yes basically even before every assert to make sure we wait for all the updates